bowtie/_report.py

```diff
@@ -327,17 +334,38 @@ def generate_badges(self, target_dir: Path, dialect: str):
                 - counts.skipped_tests
             )
             pct = (passed / total) * 100
-            impl_dir = target_dir / f"{lang}-{name}"
-            impl_dir.mkdir(parents=True, exist_ok=True)
             r, g, b = 100 - int(pct), int(pct), 0
-            badge = {
+            badge_per_draft = {
```

Comment:
This isn't per draft, is it? It's for all drafts I think.

Reply:
When I saw the issue #55 , It had the per draft compliance checked off, so I assumed that was the badge for it.

Reply:
Oh no it is per draft? I have to look at this again, I'm still confused a bit as to what directory structure you're proposing.

Reply:
As of now, the code creates the directory in the following structure:

- for per-draft-compliance:
`{path-specified-in-cli-command}/compliance/{language-implementation}/{dialect-name}`

- for supported-draft-compliance:
`{path-specified-in-cli-command}/compliance/{language-implementation}/{dialect-name}`

This file structure feels a little redundant to me, if there's a better structure in mind, please share. I'll make the changes accordingly.

Reply:
`{path-specified-in-cli-command}/{language-implementation}/compliance/{dialect-name}.json` and `{path-specified-in-cli-command}/{language-implementation}/supported-versions.json` seem less redundant to me (and implementers probably will like having all their badges in one "spot").